packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -93,34 +94,12 @@ async function fetchRoundsByTimestamp(
   }
 }
 
-function getActiveChainIds() {
-  const activeChainIds: string[] = [];
-  const isProduction = process.env.REACT_APP_ENV === "production";
-
-  for (const chainId of Object.values(ChainId)) {
-    if (!isNaN(+chainId)) {
-      continue;
-    }
-    if (
-      isProduction &&
-      [
-        ChainId.GOERLI_CHAIN_ID,
-        ChainId.FANTOM_MAINNET_CHAIN_ID,
-        ChainId.FANTOM_TESTNET_CHAIN_ID,
-      ].includes(ChainId[chainId as keyof typeof ChainId])
-    ) {
-      continue;
-    }
-    activeChainIds.push(chainId.toString());
-  }
-  return activeChainIds;
-}
+const activeChainIds = () =>
+  allChains.map((chain) => chain.id).map((chainId) => chainId.toString());
```

Comment:
```suggestion
  allChains.map((chain) => chain.id.toString());
```
more concise and less confusing imo